src/components/create-safe/status/usePendingSafeCreation.ts

```diff
@@ -46,6 +50,8 @@ export const usePendingSafeCreation = ({ txHash, setStatus }: Props) => {
   useEffect(() => {
     if (!txHash || !provider) return
 
+    trackEvent(CREATE_SAFE_EVENTS.SUBMIT_CREATE_SAFE)
```

Comment:
Similar here: This hook should only run in certain cases but I am guessing the event should be sent whenever the user submits the tx in their wallet. A better place for this would be in `safeCreationCallback`.